script/script.js

```diff
@@ -34,3 +34,63 @@ lin.addEventListener('click', () => {
   mobileMen.style.display = 'none';
   mobileNav.style.display = 'block';
 });
+
+const projects = [{
+  title: 'Keeping track of hundreds of components',
+  image: 'assets/images/Snapshoot.svg',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  language1: 'Github',
+  language2: 'Codepen',
+  language3: 'JavaScript',
+  language4: 'Bootstrap',
+  language5: 'css',
+  language6: 'Terminal',
+  language7: 'Codekit',
+  language8: 'Ruby on Rails',
```

Comment:
- [ ] **Required:** Kindly place all languages into an array and loop through them.
![image](https://user-images.githubusercontent.com/27709832/156460343-6c26451a-b551-46c8-b329-0223d56ac92f.png)

_**Hint:**_ `projects.forEach((project) => project.forEach((language) => console.log(language.language)));`

Reply:
i have used an array